packages/babel-types/src/constants.js

```diff
@@ -22,3 +22,5 @@ export const INHERIT_KEYS = {
   optional: ["typeAnnotation", "typeParameters", "returnType"],
   force: ["start", "loc", "end"]
 };
+
+export const BLOCK_SCOPED_SYMBOL = Symbol("var used to be block scoped");
```

Comment:
We'll have to use `Symbol.for("description")` here instead of `Symbol()` because we might have multiple copies of `babel-types` and we'll want to use the global symbol registry.